mobile/native/native.xcodeproj/xcuserdata/bonomat.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -7,12 +7,12 @@
 		<key>native-cdylib.xcscheme_^#shared#^_</key>
 		<dict>
 			<key>orderHint</key>
-			<integer>10</integer>
```

Comment:
🔧 I guess the bonomat.xcuserdatad shouldn't be committed.

Reply:
Sure, it's needed to run the tests 🤪

Reply:
[`4e12b91` (#1478)](https://github.com/get10101/10101/pull/1478/commits/4e12b9195227607fb452c21c6705fcd9bd49b0a6)